posthog/temporal/llm_analytics/run_evaluation.py

```diff
@@ -94,6 +97,14 @@ async def execute_llm_judge_activity(evaluation: dict[str, Any], event_data: dic
     """Execute LLM judge to evaluate the target event"""
     import openai
 
+    if evaluation["evaluation_type"] != "llm_judge" or evaluation["output_type"] != "boolean":
+        raise ValueError(f"Unsupported evaluation: {evaluation['evaluation_type']}/{evaluation['output_type']}")
```

Comment:
If you raise here, won't this activity be retried and fail again? Same for under.